packages/components/tsconfig.json

```diff
@@ -45,14 +45,11 @@
 		"src/**/stories/**/*.js", // only exclude js files, tsx files should be checked
 		"src/**/test/**/*.js", // only exclude js files, ts{x} files should be checked
 		"src/index.js",
-		"src/custom-gradient-picker",
 		"src/dimension-control",
 		"src/duotone-picker",
-		"src/gradient-picker",
```

Comment:
Removed to avoid TS errors:

```sh
packages/components/src/palette-edit/index.tsx:26:28 - error TS6307: File 'gutenberg/packages/components/src/gradient-picker/index.js' is not listed within the file list of project 'gutenberg/packages/components/tsconfig.json'. Projects must list all files or use an 'include' pattern.

26 import GradientPicker from '../gradient-picker';
```